### PR TITLE
Fixes #19781 - Fix Layout/LeadingCommentSpace cop

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

**Comment by mmoll** [comment]
Personally I prefer the style the cop enforces, but wouldn't be really sad if others want to have the cop disabled.

Opinions?

**Comment by ares** [comment]
This one is :+1: from me, but with styles cop, I'd be careful in general. It should receive multiple :+1:s before it gets in.

File: app/controllers/concerns/foreman/controller/taxonomy_multiple.rb [hunk 1]
```diff
@@ -2,8 +2,8 @@ module Foreman::Controller::TaxonomyMultiple
   extend ActiveSupport::Concern
 
   included do
-    #TODO: make this before filter work, its not working as the same filter is defined in the hosts controller
-    #before_action :find_multiple, :only => [:select_multiple_organization, :update_multiple_organization,
+    # TODO: make this before filter work, its not working as the same filter is defined in the hosts controller
+    # before_action :find_multiple, :only => [:select_multiple_organization, :update_multiple_organization,
     #                                        :select_multiple_location,     :update_multiple_location]
```
Comment (ekohl) [inline app/controllers/concerns/foreman/controller/taxonomy_multiple.rb:7]: Should this be aligned again?

File: app/services/dashboard/data.rb [hunk 2]
```diff
@@ -57,7 +57,7 @@ def fetch_data
     if out_of_sync_enabled?
       @report.update({
                        :out_of_sync_hosts         => out_of_sync_hosts.count,
-                       :out_of_sync_hosts_enabled => out_of_sync_hosts.alerts_enabled.count,
+                       :out_of_sync_hosts_enabled => out_of_sync_hosts.alerts_enabled.count
```
Comment (ekohl) [inline app/services/dashboard/data.rb]: Seems to be an unrelated fix that should be in a different commit.

File: app/controllers/concerns/foreman/controller/taxonomy_multiple.rb (hunk 1 above)
Comment (mmoll) [reply app/controllers/concerns/foreman/controller/taxonomy_multiple.rb:7]: yes. fixed.

File: app/services/dashboard/data.rb (hunk 2 above)
Comment (mmoll) [reply app/services/dashboard/data.rb]: this got in in the meanwhile, removed here.

**Comment by mmoll** [comment]
rebased.

**Review by lzap — APPROVED** [review]
I love this one.

**Comment by mmoll** [comment]
anybody? :)

**Comment by lzap** [comment]
Asked for few more formal reviews, anyone is welcome to give plus or minus here.

**Review by ekohl — APPROVED** [review]
I'm used to this because pep8 also prefers it.

**Review by tbrisker — APPROVED** [review]

**Comment by lzap** [comment]
Test failures likely unrelated.

**Review by dLobatog — APPROVED** [review]

**Comment by dLobatog** [comment]
Given the # of acks, I'm going to go with merging, thanks @mmoll
